objects/game-cheat/definition.json

```diff
@@ -0,0 +1,113 @@
+{
+  "attributes": {
+    "affected-game": {
+      "description": "Name of the game that is targeted by the cheatware.",
+      "misp-attribute": "text",
+      "ui-priority": 7
+    },
+    "cheat-name": {
+      "description": "Known name of the game cheat, if given.",
+      "disable_correlation": true,
+      "misp-attribute": "text",
+      "ui-priority": 12
+    },
+    "cheat-screenshot": {
+      "description": "Screenshot of the cheat at work.",
+      "misp-attribute": "attachment",
+      "ui-priority": 9
+    },
+    "cheat-type": {
+      "description": "Select the type of cheat.",
+      "misp-attribute": "text",
+      "multiple": true,
+      "ui-priority": 11,
+      "values_list": [
+        "Aim bot",
+        "Auto farmer",
+        "Boosting",
+        "DDoS",
+        "Disconnecting",
+        "Exploit",
+        "Fly",
+        "Force field",
+        "Ghosting",
+        "God mode",
+        "Invicibility",
+        "Macros",
+        "RapidFire",
+        "Scripting",
+        "Wallhack",
+        "Win trading",
+        "Others"
+      ]
+    },
+    "cheat-version": {
+      "description": "Any information about the cheatware version.",
+      "disable_correlation": true,
+      "misp-attribute": "text",
+      "ui-priority": 8
+    },
+    "compilation-date": {
+      "description": "Compilation date of the game cheat, if known.",
+      "disable_correlation": true,
+      "misp-attribute": "datetime",
+      "ui-priority": 5
+    },
+    "creator": {
+      "description": "Individual and/or Group and/or Organization that created the cheat.",
+      "misp-attribute": "threat-actor",
+      "ui-priority": 1
+    },
+    "ig-cheat-behaviour": {
+      "description": "Describe the in-game behaviour of the cheat (e.g. You selected 'Aim Bot', here you can add details like 'Activate by pressing F7, Deactivate by pressing F8. Not detected be Easy Anti-Cheat.'",
+      "disable_correlation": true,
+      "misp-attribute": "text",
```

Comment:
I'd pick "comment" as the attribute type here. This is a little bit of a nit-picky detail, but generally comments are best to use when the text is not meant to be correlated or used as an indicator ever but rather acts as a textual description of ongoing activities.

Reply:
I didn't know and hadn't thought about it. Done, as you said.